js/battle.js

```diff
@@ -5464,6 +5464,7 @@ var Battle = (function () {
 			this.log('<div class="debug"><div class="chat"><small style="color:#999">[DEBUG] ' + Tools.escapeHTML(name) + '.</small></div></div>', preempt);
 			break;
 		case 'unlink':
+			if (Tools.prefs('nounlinkhide')) return;
```

Comment:
``nounlink`` seems reasonable since both commands parsing ``|unlink|`` just deal with hiding messages. I shouldn't have forgotten to check that the commands work while I was focusing on writing /help instead